src/assets/javascript/images.coffee

```diff
@@ -0,0 +1,3 @@
+$ ->
+  if window.devicePixelRatio == 2
+    $("img[src$='product-stack.png']").attr('src', '/assets/images/product-stack-2x.png')
```

Comment:
This is a bummer. The SVG output from Illustrator was complicated enough to require masks. These didn't work in Safari. I expect I could spend some time and figure this out, but in the interest of time, I went with this.